java/client/test/org/openqa/selenium/virtualauthenticator/VirtualAuthenticatorTest.java

```diff
@@ -98,25 +122,58 @@ public void setup() {
         .withScripts(script)));
   }
 
+  private void createSimpleU2FAuthenticator() {
+    VirtualAuthenticatorOptions options = new VirtualAuthenticatorOptions()
+        .setProtocol(Protocol.U2F);
+    authenticator = ((HasVirtualAuthenticator) driver).addVirtualAuthenticator(options);
+  }
+
+  private void createRKEnabledAuthenticator() {
+    VirtualAuthenticatorOptions options = new VirtualAuthenticatorOptions()
+        .setProtocol(Protocol.CTAP2)
+        .setHasResidentKey(true)
+        .setHasUserVerification(true)
+        .setIsUserVerified(true);
+    authenticator = ((HasVirtualAuthenticator) driver).addVirtualAuthenticator(options);
+  }
+
+  /**
+   * @param list a list of numbers between -128 and 127.
+   * @return a byte array containing the list.
+   */
+  private byte[] convertListIntoArrayOfBytes(List<Long> list) {
+    byte[] ret = new byte[list.size()];
+    for (int i = 0; i < list.size(); ++i)
+      ret[i] = list.get(i).byteValue();
+    return ret;
+  }
+
+  private Map<String, Object> getAssertionFor(Object credentialId) {
+    return (Map<String, Object>)
+      ((JavascriptExecutor) driver).executeAsyncScript(
+        "getCredential([{"
+      + "  \"type\": \"public-key\","
+      + "  \"id\": Int8Array.from(arguments[0]),"
+      + "}]).then(arguments[arguments.length - 1]);", credentialId);
+  }
+
+  @After
+  public void tearDown() {
+    if (authenticator != null)
```

Comment:
Nit: we tend to always use braces in the selenium codebase. It makes it harder to cause chaos accidentally :)

Reply:
Fixed here and in a couple more places.